spec.html

```diff
@@ -35771,6 +36315,54 @@ <h1>
           </emu-alg>
         </emu-clause>
 
+        <emu-clause id="sec-runtime-semantics-maybesimplecasefolding-a" type="abstract operation">
+          <h1>
+            MaybeSimpleCaseFolding (
+              _A_: a CharSet,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It uses the Unicode <a href="https://www.unicode.org/reports/tr44/#Simple_Case_Folding">Simple Case Folding</a> mapping, which maps one code point to one code point, and which is abbreviated <a href="https://www.unicode.org/Public/UCD/latest/ucd/PropertyAliases.txt">scf</a>.</dd>
+          </dl>
+          <emu-alg>
+            1. If _UnicodeSets_ is *false* or _IgnoreCase_ is *false*, return _A_.
+            1. Let _S_ be a copy of _A_.
+            1. For each element _cs_ (a character or sequence of characters) in _A_, do
+              1. Create an empty sequence of characters _t_.
+              1. For each single code point _c_ in _cs_, do
+                1. Append scf(_c_) to _t_.
+              1. If _t_ is different from _cs_, remove _cs_ from _S_ and add _t_ to _S_.
+            1. Return _S_.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-runtime-semantics-getallcharacters" type="abstract operation">
+          <h1>GetAllCharacters ( )</h1>
+          <dl class="header">
+          </dl>
+          <emu-alg>
+            1. Let _A_ be a new, empty CharSet (“A” for “all characters”).
```

Comment:
```suggestion
            1. Let _A_ be a new empty CharSet.
```
Re deleting the comma, I'm using the precedent of "a new empty List".
Re deleting the parenthetical, that's not how we do comments. You could make a NOTE (or rename the alias), but I don't think it's necessary.